### PR TITLE
add GROMACS-4.6.5-intel-2015a

### PR DESCRIPTION
**Pull request by molden**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name = 'GROMACS'
+version = '4.6.5'
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb]: @molden: are you aware there's a 4.6.7 too? just wondering, I'm happy to include this one too

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bef6e73629e07e24742d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6643a986018e439fa2e4 for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb (hunk 1 above)
Comment (molden) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb]: Ok you are right, 4.6.7 should work too for g_mmpbsa. I can have a look and add the easyconfigs for Gromacs 4.6.7 and g_mmpbsa too at some point later?

Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb]: sure, this one is good to merge

**Comment by boegel** [comment]
Thanks @molden!

**Pull request merged**